tests/bears/BearTest.py

```diff
@@ -124,6 +127,20 @@ def run(self, x: int, y: int, z: int = 33):
         yield z
 
 
+class TestOneBear(Bear):
```

Comment:
Can we put these bears in `tests/test_bears/` to be re-usable.
(and add a note that the other bears here should be moved to files in that directory , and/or create an issue for someone to move them .)

Reply:
If no one solves takes the issue I will solve that :+1:

Reply:
Let's do that later, it's been done for other bears until now as well :+1: